aspnet/fundamentals/logging.rst

```diff
@@ -143,7 +143,7 @@ First, be sure to add the ``Microsoft.Framework.Logging.TraceSource`` package to
 	:lines: 5-12
 	:emphasize-lines: 8
 
-The following example demonstrates how to configure two separate ``TraceSourceLogger``s for an application, both logging only ``Critical`` messages. Each call to ``AddTraceSource`` takes a ``TraceListener``. The first call configures a ``ConsoleTraceListener``; the second one configures an ``EventLogTraceListener`` to write to the ``Application`` event log. These two listeners are not available in DNX Core, so their configuration is wrapped in a conditional compilation statement (``#if DNX451``).
+The following example demonstrates how to configure two separate ``TraceSourceLoggers`` for an application, both logging only ``Critical`` messages. Each call to ``AddTraceSource`` takes a ``TraceListener``. The first call configures a ``ConsoleTraceListener``; the second one configures an ``EventLogTraceListener`` to write to the ``Application`` event log. These two listeners are not available in DNX Core, so their configuration is wrapped in a conditional compilation statement (``#if DNX451``).
```

Comment:
Another variant could be \`\`TraceSourceLogger\`\`\s, which will be rendered as ``TraceSourceLogger``s.

I aimed for consistency :)
Look at the end of line 153:
> two ``TraceListeners``

or line 83:
> ``LogLevels``

I don't know whether there is a guideline about formatting the plurals, but as already say I would go for consistent handling of them in this article.

Which ever way is the preferred, I can update the PR.